src/python/pants/backend/debian/rules_integration_test.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+import os
+import subprocess
+import sys
+from textwrap import dedent
+
+import pytest
+
+from pants.backend.debian.rules import DebianPackageFieldSet
+from pants.backend.debian.rules import rules as debian_rules
+from pants.backend.debian.target_types import DebianPackage
+from pants.backend.python import target_types_rules
+from pants.build_graph.address import Address
+from pants.core.goals.package import BuiltPackage
+from pants.engine.rules import QueryRule
+from pants.engine.target import Target
+from pants.testutil.rule_runner import RuleRunner
+
+
+@pytest.fixture
+def rule_runner() -> RuleRunner:
+    return RuleRunner(
+        rules=[
+            *debian_rules(),
+            *target_types_rules.rules(),
+            QueryRule(BuiltPackage, (DebianPackageFieldSet,)),
+        ],
+        target_types=[DebianPackage],
+    )
+
+
+def build_package(rule_runner: RuleRunner, binary_target: Target) -> BuiltPackage:
+    field_set = DebianPackageFieldSet.create(binary_target)
+    result = rule_runner.request(BuiltPackage, [field_set])
+    rule_runner.write_digest(result.digest)
+    return result
+
+
+@pytest.mark.skipif(sys.platform != "linux", reason="Test requires dpkg so only works on Linux")
```

Comment:
Excellent point! What do you think would be be most helpful -- 
1) extend the reason message telling that you need a Debian-based Linux
2) extend the reason message telling that you need a Debian-based Linux AND do `shutil.which("dpkg")` instead of simply `sys.platform` to check if it's a Debian-based Linux?

Reply:
I thought (perhaps an overkill?) that we could actually to check for dpkg in case the tests are ever run on rpm-based Linux machine or some other Linux distro at all, so have used the `shutil.which("dpkg")` approach.